ui/src/app/utils/runtime-utils.spec.tsx

```diff
@@ -241,3 +272,895 @@ describe(maybeWithPersistentDisk.name, () => {
     );
   });
 });
+
+// diff state combination logic is covered by the findMostSevereDiffState tests
```

Comment:
These are all "descriptive" tests meant to document the existing behavior.  I made no attempt to simplify these or look for bugs - that will follow.